Makefile

```diff
@@ -78,11 +78,11 @@ lint: check-licence eclint-check
 	@go get golang.org/x/lint/golint
 	@$(foreach dir,$(PKGS),golint $(dir) 2>&1 | $(FILTER_LINT) | tee -a lint.log;)
 	@echo "Checking errcheck..."
-	@go get github.com/kisielk/errcheck
-	@errcheck $(PKGS) 2>&1 | $(FILTER_LINT) | tee -a lint.log
+	@go get github.com/kisielk/gotool
```

Comment:
Why do you need to `go get` here if you're just using `go run vendor/.../errchek`?

Reply:
actually it's `vendor/honnef.co/go/tools/cmd/staticcheck/staticcheck` has dependency on `github.com/kisielk/gotool`, not  `vendor/github.com/kisielk/errchek`

I can put this in `glide.yaml`